packages/cli/src/generator/printRunInstructions.js

```diff
@@ -13,7 +13,7 @@ import chalk from 'chalk';
 import logger from '../util/logger';
 
 function printRunInstructions(projectDir: string, projectName: string) {
-  const absoluteProjectDir = path.resolve(projectDir);
+  const relativeProjectDir = path.relative(process.cwd(), projectDir);
```

Comment:
Can we use `root` from command context?

Reply:
I think `init` doesn't have context injected, because it is called from `global-cli` - not from out `commander.js` interface.